kahuna/public/js/crop/controller.js

```diff
@@ -94,15 +102,41 @@ crop.controller('ImageCropCtrl',
         }
     });
 
-    ctrl.cropWidth = () => Math.round(ctrl.coords.x2 - ctrl.coords.x1);
-    ctrl.cropHeight = () => Math.round(ctrl.coords.y2 - ctrl.coords.y1);
+    const ratioString = (aspect) => {
+        if (Number(aspect) === ctrl.landscapeRatio) {
+            return '5:3';
+        } else if (Number(aspect) === ctrl.portraitRatio) {
+            return '4:5';
+        } else if (Number(aspect) === ctrl.squareRatio) {
+            return '1:1';
+        } else if (Number(aspect) === ctrl.videoRatio) {
+            return '16:9';
+        }
+        // else undefined is fine
+    };
+
+    ctrl.getRatioString = ratioString;
+
+    // If we have a square crop, remove any jitter introduced by client lib by using only one side
+    if(ratioString == '1:1') {
```

Comment:
The idea is to always use `===` out of superstition because JS is so fucked that it's hard to remember to do it only when it matters.